app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,18 @@
   <%= stylesheet_link_tag('application', media: 'all') %>
 <% end %>
 
+<% content_for :header_class do "with-proposition" end %>
+
+<% content_for :proposition_header do %>
+  <div class="header-proposition">
+    <div class="content">
+      <nav id="proposition-menu">
+        <a href="/" id="proposition-name"><%= t('app_title') %></a>
+      </nav>
+    </div>
+  </div>
+<% end %>
+
```

Comment:
I suppose it's not related but I noticed that the template being used was causing a problem. I will remove this and do in a separate branch.